### PR TITLE
Allow for external and inline sourcemap generation separately

### PR DESCRIPTION
**Pull request by DylanPiercey**
@michaelficarra Is this what you are talking about?

### CODE REVIEW

File: src/coffee-script.coffee [hunk 1]
```diff
@@ -95,17 +93,18 @@ exports.compile = compile = withPrettyErrors (code, options) ->
...
-      sourceURL        = "//# sourceURL=#{options.filename ? 'coffeescript'}"
-      "#{js}\n#{sourceMapDataURI}\n#{sourceURL}"
-    else 
+  if options.inlineMap
+    sourceMapDataURI = "//# sourceMappingURL=data:application/json;base64,#{base64encode v3SourceMap}"
+    sourceURL = "//# sourceURL=#{options.filename ? 'coffeescript'}"
+    js = "#{js}\n#{sourceMapDataURI}\n#{sourceURL}"
+
+  if options.sourceMap
       answer = {js}
```
Comment (michaelficarra) [inline src/coffee-script.coffee]: Please reduce the indentation of this block.

**Comment by michaelficarra** [comment]
:+1: LGTM otherwise.

File: src/coffee-script.coffee (hunk 1 above)
Comment (DylanPiercey) [inline src/coffee-script.coffee]: Was this not what you were looking for?

Comment (michaelficarra) [inline src/coffee-script.coffee]: It's fine, it's just using 4-space indent instead of the standard 2.

Comment (DylanPiercey) [inline src/coffee-script.coffee]: Right, but I did reduce it to two? Unless I'm missing something.

Comment (michaelficarra) [inline src/coffee-script.coffee]: No, look at the line I commented on.

Comment (DylanPiercey) [inline src/coffee-script.coffee]: I am blind, thanks. Fixed now.

I vote we switch to tabs :p.

**Comment by michaelficarra** [comment]
Thanks @DylanPiercey.

**Comment by DylanPiercey** [comment]
@michaelficarra No problem, good catch.

**Comment by XVincentX** [comment]
Great! I really hope to see a new release in short time!

**Comment by XVincentX** [comment]
@michaelficarra ?

**Pull request merged**